<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CSNB0366" ignored="0" name="Test" tests="6" failures="2" timestamp="02 Jan 2020 17:51:20 GMT" time="48.203" errors="0">
  <testcase name="BsecondTest" time="0.006" classname="TestNg1.TestNg1"/>
  <testcase name="AfirstTest" time="0.005" classname="TestNg1.TestNg1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNg1.TestNg1.AfirstTest(TestNg1.java:27)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- AfirstTest -->
  <testcase name="executSessionOne" time="10.186" classname="TestNg1.TestNg1"/>
  <testcase name="executeSessionTwo" time="48.134" classname="TestNg1.TestNg1">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting 45 seconds for Firefox to start.
Build info: version: &amp;apos;3.8.1&amp;apos;, revision: &amp;apos;6e95a6684b&amp;apos;, time: &amp;apos;2017-12-01T19:05:32.194Z&amp;apos;
System info: host: &amp;apos;CSNB0366&amp;apos;, ip: &amp;apos;192.168.43.47&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_161&amp;apos;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'
System info: host: 'CSNB0366', ip: '192.168.43.47', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:131)
at org.openqa.selenium.firefox.XpiDriverService.start(XpiDriverService.java:116)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:219)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:142)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
at TestNg1.TestNg1.executeSessionTwo(TestNg1.java:72)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:4527/hub/status] to be available after 45002 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:129)
... 21 more
Caused by: java.util.concurrent.TimeoutException
at java.util.concurrent.FutureTask.get(FutureTask.java:205)
at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:147)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
... 22 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- executeSessionTwo -->
  <testcase name="CthirdTest" time="0.003" classname="TestNg1.TestNg2"/>
  <testcase name="DforthTest" time="0.002" classname="TestNg1.TestNg2"/>
</testsuite> <!-- Test -->
